notebooks/forest_inference_demo.ipynb

```diff
@@ -471,7 +471,7 @@
    "outputs": [],
    "source": [
     "df = df.persist()\n",
-    "wait(df)"
+    "wait(df);"
```

Comment:
Do we want that?

Reply:
Yeah, otherwise you get a bunch of unnecessary output. The output of `wait` doesn't need to be visible and is just noise in this case.